client/cli/go/cmds/volume.go

```diff
@@ -23,20 +23,21 @@ import (
 )
 
 var (
-	size           int
-	volname        string
-	durability     string
-	replica        int
-	disperseData   int
-	redundancy     int
-	gid            int64
-	snapshotFactor float64
-	clusters       string
-	expandSize     int
-	id             string
-	kubePvFile     string
-	kubePvEndpoint string
-	kubePv         bool
+	size                 int
+	volname              string
+	durability           string
+	replica              int
+	disperseData         int
+	redundancy           int
+	gid                  int64
+	snapshotFactor       float64
+	clusters             string
+	expandSize           int
+	id                   string
+	kubePvFile           string
+	kubePvEndpoint       string
+	kubePv               bool
+	glusterVolumeOptions string
```

Comment:
Thanks for choosing a longer, more descriptive name for the options variable.
I still see a slight inconsistency: in the cli it's called glusterVolumeOptions / --gluster-volume-options, while in the api it's glusterVolOptions. Why that difference? Shouldn't we make them read the same, according to the principle of least surprise? (At least for the outside facing names cli and api.)